client/app/components/Modal/__tests__/Modal.spec.jsx

```diff
@@ -373,13 +404,13 @@ describe('Modal', () => {
         />
       );
       it('toggles correctly', () => {
-        const wrapper = mount(component);
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
-        expect(wrapper.find('.modal').exists()).toEqual(true);
-        wrapper.find('.modal').simulate('mouseover');
-        wrapper.find('.modalBackdrop').simulate('click');
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
-        expect(wrapper.find('.modal').exists()).toEqual(true);
+        const { container } = render(component);
+        expect(container.querySelector('.modalBackdrop')).toBeInTheDocument();
+        expect(container.querySelector('.modal')).toBeInTheDocument();
+
```

Comment:
Likewise, there should be a `hover` event here to simulate the `mouseover` as before. In the component, the mouseover sets the `modalHasFocus` state to `true` and prevents the modal from being closed on click.